UI/Web/src/app/manga-reader/_components/single-renderer/single-renderer.component.ts

```diff
@@ -53,6 +55,11 @@ export class SingleRendererComponent implements OnInit, ImageRenderer {
   pageNum: number = 0;
   maxPages: number = 1;
 
+  /**
+   * Width overwrite for maunal width control
+  */
+  widthOverride$ : string = 'none';
```

Comment:
The $ sign means that the variable is an observable. The idea is to map it to the widthSlider valueChanges and use map to transform it to the style you need.